target/linux/octeon/base-files/lib/upgrade/platform.sh

```diff
@@ -23,7 +23,8 @@ platform_get_rootfs() {
 
 platform_copy_config() {
 	case "$(board_name)" in
-	erlite)
+	erlite | \
+	ubnt,*)
```

Comment:
Please, no wildcards in base-files.
https://openwrt.org/docs/guide-developer/device-support-policies#no_wildcards